pandas/tests/arithmetic/test_timedelta64.py

```diff
@@ -1960,19 +1960,20 @@ def test_td64arr_floordiv_numeric_scalar(self, box_with_array, two):
             two // tdser
 
     @pytest.mark.parametrize(
-        "vector",
-        [np.array([20, 30, 40]), Index([20, 30, 40]), Series([20, 30, 40])],
-        ids=lambda x: type(x).__name__,
+        "klass",
+        [np.array, Index, Series],
```

Comment:
we have a fixture for this already no? Can't we just use that?

Reply:
We don't have one yet like this; only ones that parametrize over pandas constructors. I do think we should have a "1D constructor" fixture in the future